modules/imaging_browser/test/imaging_browserTest.php

```diff
@@ -242,87 +242,86 @@ public function setUp(): void
         // @codingStandardsIgnoreStart
         $this->DB->insert(
             'files',
-            array(
-             'FileID'                => 1111,
-             'SessionID'             => 999998,
-             'File'                  => 'assembly/506145/V1/mri/native/' .
-              'loris-MRI_506145_V1_t2_001.mnc',
-             'SeriesUID'             => '1.3.12.2.1107.5.2.32.35049.' .
-               '2014021711090977356751313.0.0.0',
-             'EchoTime'              => 0.011,
-             'CoordinateSpace'       => 'native',
-             'OutputType'            => 'native',
-             'AcquisitionProtocolID' => 45,
-             'FileType'              => 'mnc',
-             'InsertedByUserID'      => 'lorisadmin',
-             'InsertTime'            => 1454951768,
-             'SourcePipeline'        => null,
-             'PipelineDate'          => null,
-             'SourceFileID'          => 1111,
-             'ProcessProtocolID'     => 1111,
-             'Caveat'                => 0,
-             'TarchiveSource'        => 263,
-            )
+            [
```

Comment:
I don't think this file should show up in the diff of that PR. Everything seems to be changes from `array()` to `[]`. I don't remember if there is a LORIS convention on how to encode those but in any case, it should not show up in this PR.

Reply:
This is a remnant of the original PR, removed.